### PR TITLE
Logic phenotype

### PR DESCRIPTION
**Pull request by a-hartens**
…lemented yet

### CODE REVIEW

File: phenex/phenotypes/computation_graph_phenotypes.py [hunk 1]
```diff
@@ -96,6 +114,54 @@ def _execute(self, tables: Dict[str, Table]) -> PhenotypeTable:
...
+
+        # do the union of all the non-null dates
+        all_dates = non_null_dates_by_date_col[0]
+        for non_null_dates in non_null_dates_by_date_col[1:]:
+            all_dates = all_dates.union(non_null_dates)
+        return all_dates
+
+    def _coalesce_all_date_columns(self, table):
+        """
+        Generate COALESCE strings for the GREATEST or LEAST function using Ibis.
```
Comment (sprivite) [inline phenex/phenotypes/computation_graph_phenotypes.py:145]: are we back to working with strings ... ?

**Review by sprivite — COMMENTED** [review]
lgtm except the one comment

File: phenex/phenotypes/computation_graph_phenotypes.py (hunk 1 above)
Comment (a-hartens) [reply phenex/phenotypes/computation_graph_phenotypes.py:150]: nope, it uses ibis but is doing coalesce... changed docstring

Comment (sprivite) [reply phenex/phenotypes/computation_graph_phenotypes.py]: Awesome

**Pull request merged**